models/spike_recorder.h

```diff
@@ -64,6 +64,34 @@ spike creation rather than that of their arrival.
 The call to ``Connect`` will fail if the connection direction is
 reversed (i.e., connecting *sr* to *neurons*).
 
+Properties
+++++++++++
+
+All recorders have a set of common properties that can be set using
+``SetDefaults`` on the model class or ``SetStatus`` on a device instance:
+
+===========  ======================================================================
+ label        A string (default: `“”`) specifying an arbitrary textual label for
```

Comment:
I'm all for UTF-8, but it's probably safer to stick to ascii here.
```suggestion
 label        A string (default: `""`) specifying an arbitrary textual label for
```